easybuild/easyconfigs/c/canu/canu-1.7-foss-2018b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'MakeCp'
+
+name = 'canu'
+version = '1.7'
+
+homepage = 'http://canu.readthedocs.io'
+description = """Canu is a fork of the Celera Assembler designed for
+ high-noise single-molecule sequencing"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/marbl/canu/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['c5be54b0ad20729093413e7e722a19637d32e966dc8ecd2b579ba3e4958d378a']
+
+dependencies = [('Boost', '1.68.0', '-Python-2.7.15')]
```

Comment:
@fizwit Please stick to Boost 1.67.0 in the 2018b generation of easyconfigs (see failing test).